adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala

```diff
@@ -282,18 +281,35 @@ case class ParquetUnboundFeatureRDD private[rdd] (
     sqlContext.read.parquet(parquetFilename).as[FeatureProduct]
   }
 
+  /**
+   * Replaces the sequence dictionary attached to a ParquetUnboundFeatureRDD.
+   *
+   * @param newSequences The new sequence dictionary to attach.
+   * @return Returns a new ParquetUnboundFeatureRDD with the sequences replaced.
+   */
   def replaceSequences(newSequences: SequenceDictionary): FeatureRDD = {
```

Comment:
This method is defined with scaladoc in `GenomicDataset`, could we use `override def` instead?